tofhir-server/src/main/scala/io/tofhir/server/service/ExecutionService.scala

```diff
@@ -103,9 +102,16 @@ class ExecutionService(jobRepository: IJobRepository, mappingRepository: IMappin
     }
 
     // create execution
-    val mappingJobExecution = FhirMappingJobExecution(executionId.getOrElse(UUID.randomUUID().toString), job = mappingJob, projectId = projectId, mappingTasks = mappingTasks,
-      mappingErrorHandling = executeJobTask.flatMap(_.mappingErrorHandling).getOrElse(mappingJob.dataProcessingSettings.mappingErrorHandling))
-    val fhirMappingJobManager = getFhirMappingJobManager(mappingJob.dataProcessingSettings.mappingErrorHandling)
+    val mappingJobExecution = FhirMappingJobExecution(executionId.getOrElse(UUID.randomUUID().toString), job = mappingJob,
+      projectId = projectId, mappingTasks = mappingTasks)
+    val fhirMappingJobManager = new FhirMappingJobManager(
```

Comment:
Let's make it a ExecutionService field so that we can utilize it in both functions.

Reply:
When I make it an ExecutionService field, It causes a null pointer error in FhirMappingJobManager#62. It is thrown inside of a library function, and I did not find any reason for this behavior.